docker/.env.example.full

```
@@ -642,9 +642,9 @@ STRIPE_WECHAT_PAY_ENABLED="False"
 # Type: int
 REDIS_DB="0"
 
-# Redis server host (docker-compose service name by default)
-# (Optional - default: ai-shifu-redis)
-REDIS_HOST="ai-shifu-redis"
+# Redis server host (optional; leave empty to disable Redis)
+# (Optional - default: )
+REDIS_HOST=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)